about.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Page</title>
+    <script
+      src="https://kit.fontawesome.com/58789452ae.js"
+      crossorigin="anonymous">
+    </script>
+    <link rel="stylesheet" href="about.css">
+  </head>
+
+  <body class="homepage">
+    <header class="nav-bar" id="nav-bar"> 
+      <div class="intro-part">
+        <button class="mobile" onclick="openMobileMenu()">
+          <img class="toggl" src="./image/mobile_menu.png" alt="menu icon">
+        </button>
+        <div class="desktop">
+          <div class="top-nav">
+            <ul class="toplist">
+              <li>
+                <a href="#">
+                  <i class="fa-brands fa-facebook top-item faceb"></i>
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com/QueenJumah">
+                  <i class="fa-brands fa-twitter top-item faceb"></i>
+                </a>
+              </li>
+              <li class="top-item">English</li>
+              <li class="top-item">Mypage</li>
+              <li class="top-item">Logout</li>
+            </ul>
+          </div>
+          <div class="navdesktop">
+            <div class="logo">
+              <a href="index.html"><img class = "logo-img" src="./image/mylogo.png" alt="the royal dev logo"/></a>
+            </div>
+            <nav class="navigation">
+              <ul class="navigation-item">
+                <li class="nav-item"><a class="sub-item abot" href="about.html">About</a></li>
+                <li class="nav-item"><a class="sub-item" href="#Program">Program</a></li>
+                <li class="nav-item"><a class="sub-item" href="#Join">Join</a></li>
+                <li class="nav-item"><a class="sub-item" href="#Sponsor">Sponsor</a></li>
+                <li class="nav-item"><a class="sub-item" href="#News">News</a></li>
+              </ul>
+              <a href="index.html"><button class="nav-works">@TRIC</button></a>
+            </nav>
+          </div>
+        </div>
+      </div>
+      <!-- popup section -->
+      <nav class="navmobile" id="navmobile">
+        <div class="btn-wrapper">
+            <button class="close-btn" onclick="closeMobileMenu()">
+              <img src="./image/close_mobile_menu.png" alt="closebtn">
+            </button>
+        </div>
+        <div class="logo" onclick="closeMobileMenu()">
+          <a href="home.html"><img class = "logo-img mobilelog" src="./image/mylogo.png" alt="the royal dev logo"/></a>
```

Comment:
Kindly fix the href attribute since we have no route like home.html. It needs to go to the index.html 🌐 
![Screen Recording 2023-02-07 at 7 51 10 PM](https://user-images.githubusercontent.com/25887253/217324722-2c7e6bb0-624c-4982-aae0-ddb150f7b0b0.gif)